ext/pgsql/pgsql.c

```diff
@@ -2876,7 +2879,8 @@ PHP_FUNCTION(pg_lo_import)
 		CHECK_DEFAULT_LINK(link);
 	}
 	else {
-		WRONG_PARAM_COUNT;
+		zend_wrong_param_count();
+		RETURN_THROWS();
 	}
```

Comment:
I think this can just be fixed by first checking for the deprecated signature with the QUIET flag, and then checking for the  signature without the quiet flag.